install/syndesis.yml

```diff
@@ -1629,6 +1648,16 @@ objects:
   message: |-
     Syndesis upgrade process started
   objects:
+  - apiVersion: v1
+    kind: PersistentVolumeClaim
```

Comment:
@nicolaferraro one last question: What happens when this PVC already exists ? which is likely as it has been created from the beginning. Does the update action just ignore it then ?